src/core/scene/a-scene.js

```diff
@@ -110,10 +113,10 @@ var AScene = module.exports = registerElement('a-scene', {
     enterVR: {
       value: function (event) {
         this.setStereoRenderer();
-        if (isMobile) {
```

Comment:
this is unrelated to the mobile regression (#1052), but this was wrong to assume. we can't assume that the WebVR API isn't going to be on mobile (the old WebVR API has been available for Firefox for Android and Chrome for Android [dev channel] - and now Brandon has new experimental Android builds for the WebVR 1.0 API).